package.json

```diff
@@ -15,8 +15,14 @@
     "@commitlint/config-conventional": "^7.5.0",
     "@commitlint/travis-cli": "^7.5.0",
     "@types/mocha": "^5.0.0",
+    "@typescript-eslint/eslint-plugin": "^1.4.2",
+    "@typescript-eslint/parser": "^1.4.2",
     "coveralls": "^3.0.0",
     "cz-conventional-changelog": "^2.1.0",
+    "eslint": "^5.14.1",
+    "eslint-config-prettier": "^4.1.0",
+    "eslint-plugin-eslint-plugin": "^2.0.1",
+    "eslint-plugin-mocha": "^5.3.0",
```

Comment:
I think these new dependencies at root level should not be needed as long as we always use the eslint config bundled as part of `@loopback/build`. Could you please try to remove them?